convex/internal/newsletter.ts

```
@@ -142,17 +143,23 @@ export const getQuestionForUser = internalAction({
 			questionId: question._id,
 		});
 
-		const baseUrl = process.env.NEXT_PUBLIC_APP_URL || "https://breaktheiceberg.com/";
-
-
+		const baseUrl = `${process.env.NEXT_PUBLIC_APP_URL || "https://breaktheiceberg.com"}/`
+			.replace(/\/{2,}$/, "/");
```

ESLint

[error] Move this regular expression to module scope to avoid re-compilation on every call.

(e18e/prefer-static-regex)